standard/template/sections/clause_7_normative_text.adoc

```diff
@@ -412,16 +412,26 @@ TIP: might be useful to describe or provide extension for referencing GeoZarr as
 
 == Annex B: Mappings with other formats
 
-TIP: Provides the mappings for information purpose to show how GEoZarr preserve information from any data source.
-
-To ensure interoperability with most client and mapping tools, GeoZarr enforces a set of requirements, including conventions from CF and GDAL.
+TIP: Provides the mappings for information purpose to show how source formats can preserve information from any data source.
 
 To maximize compatibility with various source formats, GeoZarr preserves as much metadata and structure as possible from these formats.
 
 NOTE: In particular, if relevant information which cannot be encoded in GeoZarr is identified, the specification might be extended.
 
-
 === Mappings with CF
 
-=== Mappings with GDAL
+
+=== Mappings with GeoTiff
+
+To map a GeoTIFF to the GeoZarr structure, we need to carefully translate the data arrays, coordinate variables, and metadata (such as the CRS) into the appropriate GeoZarr elements. 
+
+GeoZarr is structured as a single GeoZarr dataset at the root, encapsulating all necessary components to represent the geospatial data and metadata effectively.
+
+- GeoTIFF Data Array to GeoZarr Data Variable: In the case of a single band, the data variable represents a 2D raster with latitude and longitude dimensions. If there are multiple bands, they might be mapped to positions within a band dimension, with coordinates providing the wavelength and standard names indicating the units of measure for those coordinates.
```

Comment:
does this have to be specific to latitude and longitude?   there shouldn't be any assumption about what a GeoTIFF represents, it will be paired coords for sure but will depend on the CRS and what it dictates.

Reply:
No it shoud not. As you understood, I'm focusing on the Table of Content, and I totally understand that some text for illustration purpose may confuse people expert in a domain.